modules/pipeline/pipengine/reconciler/dbgc.go

```diff
@@ -35,6 +36,88 @@ const (
 	etcdDBGCDLockKeyPrefix = "/devops/pipeline/dbgc/dlock/"
 )
 
+func (r *Reconciler) PipelineDatabaseGC() {
```

Comment:
Add function comment and explain the difference between `ListenDatabaseGC` and `EnsureDatabaseGC`.

Reply:
Add comment for all functions.